docs/manuals/core/concepts/job.md

```diff
@@ -1,5 +1,5 @@
-Tasks, Pipelines and Scenarios entities can be submitted for execution. The submission of a scenario triggers the
-submission of all the contained pipelines. Similarly, submission of a pipeline triggers the submission of
+Tasks, Sequences and Scenarios entities can be submitted for execution. The submission of a scenario triggers the
+submission of all the contained tasks. Similarly, submission of a pipeline also triggers the submission of
 all the contained tasks.
```

Comment:
```suggestion
Tasks, Sequences, and Scenarios entities can be submitted for execution. The submission of a scenario triggers the
submission of all the contained tasks. Similarly, the submission of a pipeline also triggers the request of
all the ordered tasks.
```

Reply:
@erikfTW I think `request` is not the accurate word though. Can we keep `submission` or change it to `execution`?

Reply:
> @erikfTW I think `request` is not the accurate word though. Can we keep `submission` or change it to `execution`?

Right, I would go for execution if possible.